intf/vernacexpr.ml

```diff
@@ -279,22 +279,7 @@ type bullet =
 
 (** {6 Types concerning the module layer} *)
 
-(** Rigid / flexible module signature *)
-
-type 'a module_signature =
-  | Enforce of 'a (** ... : T *)
-  | Check of 'a list (** ... <: T1 <: T2, possibly empty *)
-
-(** Which module inline annotations should we honor,
-    either None or the ones whose level is less or equal
-    to the given integer *)
-
-type inline =
```

Comment:
Why not keep these as deprecated aliases?

Reply:
well-spotted merci.